EMS/src/Entities/Question.java

```diff
@@ -8,29 +8,43 @@
 import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
-import java.sql.Timestamp;
 import java.util.Vector;
 
 /**
```

Comment:
Missing class documentation

Reply:
The class is updated now and the documentation is **added**.